SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java

```diff
@@ -25,65 +27,34 @@ public class SEPADirectDebitClient {
      */
-    public SEPADirectDebitClient(@NonNull FragmentActivity activity,
-                                 @NonNull BraintreeClient braintreeClient) {
-        this(activity, activity.getLifecycle(), braintreeClient,
-                new SEPADirectDebitApi(braintreeClient));
-    }
-
-    /**
-     * Create a new instance of {@link SEPADirectDebitClient} from within a Fragment using a
-     * {@link BraintreeClient}.
-     *
-     * @param fragment        an Android Fragment
-     * @param braintreeClient a {@link BraintreeClient}
-     */
-    public SEPADirectDebitClient(@NonNull Fragment fragment,
-                                 @NonNull BraintreeClient braintreeClient) {
-        this(fragment.getActivity(), fragment.getLifecycle(), braintreeClient,
-                new SEPADirectDebitApi(braintreeClient));
+    public SEPADirectDebitClient(@NonNull BraintreeClient braintreeClient) {
+        this(braintreeClient, new SEPADirectDebitApi(braintreeClient));
     }
 
     @VisibleForTesting
-    SEPADirectDebitClient(FragmentActivity activity, Lifecycle lifecycle,
-                          BraintreeClient braintreeClient, SEPADirectDebitApi sepaDirectDebitApi) {
-        this.sepaDirectDebitApi = sepaDirectDebitApi;
+    SEPADirectDebitClient(BraintreeClient braintreeClient, SEPADirectDebitApi sepaDirectDebitApi) {
         this.braintreeClient = braintreeClient;
-        if (activity != null && lifecycle != null) {
-            SEPADirectDebitLifecycleObserver observer = new SEPADirectDebitLifecycleObserver(this);
-            lifecycle.addObserver(observer);
-        }
-    }
-
-    /**
-     * Add a {@link SEPADirectDebitListener} to your client to receive results or errors from the
-     * SEPA Direct Debit flow.
-     *
-     * @param listener a {@link SEPADirectDebitListener}
-     */
-    public void setListener(SEPADirectDebitListener listener) {
-        this.listener = listener;
+        this.sepaDirectDebitApi = sepaDirectDebitApi;
     }
 
     /**
-     * Initiates a browser switch to display a mandate to the user. Upon successful mandate
-     * creation, tokenizes the payment method and returns a result to the
-     * {@link SEPADirectDebitListener}.
+     * Starts the SEPA tokenization process by creating a {@link SEPADirectDebitResponse} to be used
+     * to launch the SEPA mandate flow in
+     * {@link SEPADirectDebitLauncher#launch(FragmentActivity, SEPADirectDebitResponse)}
      *
-     * @param activity               an Android FragmentActivity
-     * @param sepaDirectDebitRequest the {@link SEPADirectDebitRequest}.
+     * @param activity Android FragmentActivity
+     * @param sepaDirectDebitRequest {@link SEPADirectDebitRequest}
+     * @param callback {@link SEPADirectDebitFlowStartedCallback}
      */
-    public void tokenize(final FragmentActivity activity,
-                         final SEPADirectDebitRequest sepaDirectDebitRequest) {
+    public void tokenize(@NonNull final FragmentActivity activity,
```

Comment:
Can activity be removed from this method? Doesn't look like it's used anymore

Reply:
Done!